source/a_box/tmu/tmu.c

```diff
@@ -132,7 +132,7 @@ void readTemps(tmu_handle_t *tmu) {
         SEND_MOD_CELL_TEMP_MAX(q_tx_can, tmu->tmu1_max, tmu->tmu2_max, tmu->tmu3_max, tmu->tmu4_max);
         SEND_MOD_CELL_TEMP_MIN(q_tx_can, tmu->tmu1_min, tmu->tmu2_min, tmu->tmu3_min, tmu->tmu4_min);
         uint16_t max_temp = MAX(MAX(tmu->tmu1_max, tmu->tmu2_max), MAX(tmu->tmu3_max, tmu->tmu4_max));
```

Comment:
why is min uint16 and max int16

Reply:
I have rocks in my brain that's why, switched both to int16